.gitignore

```diff
@@ -302,7 +302,7 @@ build/
 /cmake-build-debug
 
 #Doxygen
-[Dd]ocs/
+/[Dd]ocs/
```

Comment:
Can we keep `[Dd]ocs/` as well since that is what running `doxygen Doxyfile` at the root uses?

Reply:
The forward slash at the beginning targets the docs folder at the root of the repo. It also enables me to edit files in `/eng/docs` without having to force add every file that gets changed.